aws/lambda/nodejs-graphql/waypoint.hcl

```diff
@@ -0,0 +1,31 @@
+project = "waypoint-apollo-lambda"
+
+app "waypoint-apollo-lambda" {
+  build {
+    use "docker" {
+      buildkit = true
+      platform = "amd64"
+    }
+
+    registry {
+      use "aws-ecr" {
+        region     = "us-east-1"
+        repository = "waypoint-apollo-lambda"
+        tag        = gitrefpretty()
+      }
+    }
+  }
+
+  deploy {
+    use "aws-lambda" {
+      region = "us-east-1"
+      memory = 256
+    }
+  }
+
+  release {
+    use "aws-alb" {
+
+    }
```

Comment:
Small nit, but could we remove the extra space and or line the brackets? `use "aws-alb" { } `

Reply:
Yeah... I was actually staring at this, and hoping `waypoint fmt` would _fix it 🪄_... but it didn't so I carried on 🤷 ...

Fix incoming!